src/main/scala/it/pagopa/interop/backendforfrontend/api/impl/EServicesApiServiceImpl.scala

```diff
@@ -893,25 +893,28 @@ final case class EServicesApiServiceImpl(
           "voucherLifespan"         -> Json.fromInt(descriptor.voucherLifespan),
           "dailyCallsPerConsumer"   -> Json.fromInt(descriptor.dailyCallsPerConsumer),
           "dailyCallsTotal"         -> Json.fromInt(descriptor.dailyCallsTotal),
-          "description"             -> descriptor.description.map(Json.fromString).getOrElse(Json.obj()),
-          "agreementApprovalPolicy" -> Json.fromString(descriptor.agreementApprovalPolicy.toString)
+          "description"             -> descriptor.description.map(Json.fromString).getOrElse(Json.Null),
+          "agreementApprovalPolicy" -> Json.fromString(descriptor.agreementApprovalPolicy.toApi.toString)
         ),
         "riskAnalysis" -> Json.arr(
           eService.riskAnalysis.map(ra =>
             Json.obj(
-              "version"       -> Json.fromString(ra.riskAnalysisForm.version),
-              "singleAnswers" -> Json.arr(
-                ra.riskAnalysisForm.singleAnswers.map(sa =>
-                  Json.obj(
-                    "key"   -> Json.fromString(sa.key),
-                    "value" -> sa.value.map(Json.fromString).getOrElse(Json.obj())
-                  )
-                ): _*
-              ),
-              "multiAnswers"  -> Json.arr(
-                ra.riskAnalysisForm.multiAnswers.map(ma =>
-                  Json.obj("key" -> Json.fromString(ma.key), "values" -> Json.arr(ma.values.map(Json.fromString): _*))
-                ): _*
+              "name"             -> Json.fromString(ra.name),
+              "riskAnalysisForm" -> Json.obj(
+                "version"       -> Json.fromString(ra.riskAnalysisForm.version),
+                "singleAnswers" -> Json.arr(
+                  ra.riskAnalysisForm.singleAnswers.map(sa =>
+                    Json.obj(
+                      "key"   -> Json.fromString(sa.key),
+                      "value" -> sa.value.map(Json.fromString).getOrElse(Json.Null)
```

Comment:
same

Reply:
[Same](https://github.com/pagopa/interop-be-backend-for-frontend/pull/289/files#r1660596865)